code/src/App.js

```diff
@@ -1,9 +1,16 @@
-import React from 'react'
+import React from "react"
+
+import { Header } from "./Header"
+import { Form } from "./Form"
+import { Footer } from "./Footer"
+import "index.css"
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+      <Header />
+      <Form />
+      <Footer />
+    </>
   )
 }
```

Comment:
Great that you keep the App.js so clean and use components for header, form and footer!